files/docker/node/alpha/dockerfile_stage3alpha

```diff
@@ -107,7 +107,11 @@ ADD https://hydra.iohk.io/build/5102327/download/1/launchpad-db-sync-config.json
 ADD https://raw.githubusercontent.com/cardano-community/guild-operators/alpha/files/docker/node/addons/banner.txt /home/guild/.scripts/
 ADD https://raw.githubusercontent.com/cardano-community/guild-operators/alpha/files/docker/node/addons/master-topology.sh /home/guild/.scripts/
 ADD https://raw.githubusercontent.com/cardano-community/guild-operators/alpha/files/docker/node/addons/block_watcher.sh /home/guild/.scripts/
+<<<<<<< HEAD
+ADD https://raw.githubusercontent.com/cardano-community/guild-operators/alpha/files/docker/node/addons/healthcheck.sh /home/guild/.scripts/
+=======
 ADD https://raw.githubusercontent.com/cardano-community/guild-operators/alpha/files/docker/nodlaze/addons/healthcheck.sh /home/guild/.scripts/
+>>>>>>> origin/alpha
```

Comment:
Remove `>>>>>>> origin/alpha`

Reply:
Not sure what you are saying... can you explain?

Reply:
This line `>>>>>>> origin/alpha` is now part of your `files/docker/node/alpha/dockerfile_stage3alpha` file inside the local github branch. You were to delete all content from the merge menu that you did not want and only leave the lines you did want. 

Unfortunately the way merge resolution looks is like it knows what you should do and would do it when you click to resolve. This is now how it works, instead it is all text put into your file and you need to edit the file to look how you want and then click resolve.